tests/components/advantage_air/test_climate.py

```diff
@@ -97,6 +115,8 @@ async def test_climate_async_setup_entry(hass, aioclient_mock):
     assert len(aioclient_mock.mock_calls) == 9
     assert aioclient_mock.mock_calls[-2][0] == "GET"
     assert aioclient_mock.mock_calls[-2][1].path == "/setAircon"
+    data = loads(aioclient_mock.mock_calls[-2][1].query["json"])
+    assert data["ac1"]["info"]["setTemp"] == 25
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6333224)